packages/react-scripts/scripts/init.js

```diff
@@ -125,6 +125,27 @@ module.exports = function(
 
   const templatePackage = templateJson.package || {};
 
+  // TODO: Deprecate support for root-level `dependencies` and `scripts` in v4.
```

Comment:
Should this say v5 then?

Reply:
I'll merge this, and fix in the next PR (which is about deprecations) - thanks!